.github/workflows/ci-core.yml

```diff
@@ -1,6 +1,10 @@
 name: CI
 
-on: [push]
+on: 
+  push:
+    paths:
+      - '*/**'
+      - '!contracts/**'
```

Comment:
Ditto

Reply:
*/** will include the root yarn.lock for us here